.github/commit_summarizer.yaml.txt

```diff
@@ -1,17 +0,0 @@
-name: GPT Commits summarizer
```

Comment:
GPT summary of [d2e715](https://github.com/SampatSuthar/ml-demo/blob/f4c31c4d6e1e13ad49b9c7c04ec99df1d76c930c/.github/commit_summarizer.yaml.txt#d2e715e39db0ed235fcb2ad065c23b2328a31106) - [d2e715](https://github.com/SampatSuthar/ml-demo/blob/3b5f0c91ad8d16afbc5e16fd55fcf18cccb850c8/.github/commit_summarizer.yaml.txt#d2e715e39db0ed235fcb2ad065c23b2328a31106):
* Removed name and summary from .github/commit_summarizer.yaml
* Updated pull request event type to "opened" and "synchronize"
* Added job to run on ubuntu-latest
* Added action to use KanHarI/gpt-commit-summarizer@master
* Added environment variables for GITHUB_TOKEN and OPENAI_API_KEY